web/src/Pages/MyWords.elm

```diff
@@ -0,0 +1,306 @@
+        Just studentId ->
+            Api.getDetailed
+                (Endpoint.myWords
+                    (Config.restApiUrl config)
+                 -- studentId
+                )
+                (Session.cred session)
+                GotMyWords
+                -- TODO: change the decoder
+                myWordsDecoder
+
+        Nothing ->
+            Cmd.none
+
+
+
+-- DECODE
+-- TODO: make this a single decoder
+
+
+myWordsDecoder : Json.Decode.Decoder (List MyWordsItem)
+myWordsDecoder =
+    Json.Decode.list myWordItemDecoder
+
+
+myWordItemDecoder : Json.Decode.Decoder MyWordsItem
+myWordItemDecoder =
+    Json.Decode.succeed MyWordsItem
+        |> required "phrase" Json.Decode.string
+        |> required "context" Json.Decode.string
+        |> required "lemma" Json.Decode.string
+        |> required "translation" Json.Decode.string
+
+
+
+-- SHARED
+
+
+save : SafeModel -> Shared.Model -> Shared.Model
+save (SafeModel model) shared =
+    shared
+
+
+load : Shared.Model -> SafeModel -> ( SafeModel, Cmd Msg )
+load shared (SafeModel model) =
+    ( SafeModel
+        { model
+            | profile = Profile.toStudentProfile shared.profile
+        }
+    , case StudentProfile.studentID (Profile.toStudentProfile shared.profile) of
+        Just id ->
+            if id /= 0 then
+                updateMyWords
+                    shared.session
+                    shared.config
+                    (Profile.toStudentProfile shared.profile)
+
+            else
+                Cmd.none
+
+        Nothing ->
+            Cmd.none
+    )
+
+
+subscriptions : SafeModel -> Sub Msg
+subscriptions (SafeModel model) =
+    Sub.none
+
+
+
+-- VIEW
+
+
+view : SafeModel -> Document Msg
+view (SafeModel model) =
+    { title = "MyWords"
+    , body =
+        [ div [ id "my-words-box" ]
+            [ div [ id "my-words-title" ] [ text "My Words" ]
+            , div []
+                [ viewContent (SafeModel model) ]
+            ]
+        ]
+    }
+
+
+viewContent : SafeModel -> Html Msg
+viewContent (SafeModel model) =
+    div []
+        [ viewMyWordsIntro
+        , viewTable model.myWords -- TODO: put a `List (MyWordsItem)` here
+        ]
+
+
+viewMyWordsIntro : Html Msg
+viewMyWordsIntro =
+    div [ id "my-words-intro" ]
+        [ Html.text "These are the words that have been saved to My Words." ]
+
+
+viewTable : List MyWordsItem -> Html Msg
+viewTable myWords =
+    div [ id "my-words-table" ]
+        [ table [] <|
+            [ tr []
+                [ th [] [ text "Phrase" ]
+                , th [] [ text "Context" ]
+                , th [] [ text "Dictionary Form" ]
+                , th [] [ text "Translation" ]
+                ]
+            ]
+                -- for every item in myWords make a viewMyWordsRow
+                ++ List.map viewMyWordsRow myWords
+
+        --   div [ id "my-words" ] (List.map viewMyWordsRow myWords)
+        ]
+
+
+
+-- viewMyWordsRow : MyWordsItem -> List (Html msg)
```

Comment:
Looks like a leftover comment here. Let's clean this up. 

(Also a couple more a few lines down.)